gradio/data_classes.py

```diff
@@ -164,6 +164,7 @@ class FileData(GradioModel):
     orig_name: Optional[str] = None  # original filename
     mime_type: Optional[str] = None
     is_stream: bool = False
+    meta: dict = {"_type": "gradio.FileData"}
```

Comment:
Yea originally I had it

* meta: "gradio.FileData"

but felt that it didn't signal "internal use" clearly. I also considered

* "_type": "gradio.FileData"

but pydantic ignores props that begin with "_" when dumping to a json.

So I figured this was the clearest and sufficiently "unique" that it would be unlikely to cause issues down the line